uui-components/src/pickers/PickerToggler.tsx

```diff
@@ -144,7 +153,13 @@ function PickerTogglerComponent<TItem, TId>(props: PickerTogglerProps<TItem, TId
         [props.isOpen, props.closePickerBody],
     );
 
-    const icon = props.icon && <IconContainer icon={ props.icon } onClick={ props.onIconClick } />;
+    const icon = props.icon && (
+        <IconContainer
+            icon={ props.icon }
+            onClick={ props.onIconClick }
+            rawProps={ { 'data-testid': 'uui-PickerToggler-iconContainer' } }
```

Comment:
is it possible to use only aria-label & aria-role=button?

Reply:
How will you do it and why should you do that? This icon is custom, can be ignored by the user, and can be added to the picker input with some purpose (unknown). Why should we look for an icon in the array of other icons with some `aria-label`, which is not correspond to the real purpose of the button just for testing needs?